### ISSUE TITLE
Allow users to change the push-to-talk button

### ISSUE THREAD

**Issue by amnstring** [collant/vam-orchestrator#2]
Please consider adding a configuration feature to allow a user to change the push-to-talk button to something else. Most users use the right mouse and the controller trigger for lots of other things.

**Comment by collant** [comment]
Yes, actually, I can let users choose any button for that, will integrate this into the next version.

There will be two configurations, one for Desktop and one for VR.

On the first version, I didn't realize people user their desktop often and didn't include support for the keyboard.

Also, it would be interesting to map other functionnalities to the keyboard buttons.
